microprofile.ls/README.md

```diff
@@ -134,6 +134,28 @@ In other words, `mp.messaging.outgoing.generated-price.topic` exists only
 if there is the declaration `mp.messaging.outgoing.generated-price.connector=smallrye-kafka` 
 
 The `mp.messaging.outgoing.generated-price.topic` property cannot be computed on the external component side because it depends on the value of 
-`mp.messaging.outgoing.generated-price.connector`. The comput	tion is done on MicroProfile LS side with custom builder by using Java SPI [ItemMetadataProviderFactory](/src/main/java/org/eclipse/lsp4mp/extensions/ItemMetadataProviderFactory).
+`mp.messaging.outgoing.generated-price.connector`. The computation is done on MicroProfile LS side with custom builder by using Java SPI [ItemMetadataProviderFactory](/src/main/java/org/eclipse/lsp4mp/extensions/ItemMetadataProviderFactory).
```

Comment:
This link is broken (should have `org.eclipse.lsp4mp.ls` at the beginning of the path)

Reply:
fixed